src/zodbupdate/serialize.py

```diff
@@ -87,6 +88,16 @@ def find_module(self, fullname, path=None):
             return self
         return None
 
+    def find_spec(self, fullname, path=None, target=None):
+        # Python 3.12+ requires to implement this method, it no longer does
+        # the fallback on its own.
+        # See
+        # https://docs.python.org/3/library/importlib.html#importlib.abc.MetaPathFinder.find_spec
+        loader = self.find_module(fullname, path)
+        if loader is None:
+            return None
+        return importlib.util.spec_from_loader(fullname, self)
```

Comment:
This is the important part which was needed to get it running with Python 3.12+.
I transferred the code here from https://github.com/python/cpython/blob/ab4746261870bcdbb6e7c5a5b307465fbe517790/Lib/importlib/_bootstrap.py#L1044-L1051 which was called up to Python 3.11.